Makefile

```diff
@@ -184,7 +184,7 @@ test-unit: generate fmt vet manifests ## Run Unit tests.
 # 3) Timeout waiting for process kube-apiserver to stop https://github.com/kubernetes-sigs/controller-runtime/issues/1571
 test-integration: export USE_EXISTING_CLUSTER=true
 test-integration: generate fmt vet manifests ## Run Integration tests.
-	go test ./... -tags integration -ginkgo.v -ginkgo.progress -v -timeout 600s
```

Comment:
consider using ginkgo CLI

I changed it recently in another operator https://github.com/Kuadrant/kuadrant-operator/pull/425/commits/f3553fa525d3e1b84c39f8533eecbdf22b5716a6

Reply:
@eguzki What's the advantage of using the ginkgo CLI?

Reply:
Nothing really important that we need not. But reading the [doc](https://onsi.github.io/ginkgo/)

```
Under the hood, ginkgo is simply calling go test. While you can run go test instead of the ginkgo CLI, Ginkgo has several capabilities that can only be accessed via ginkgo. We generally recommend users embrace the ginkgo CLI and treat it as a first-class member of their testing toolchain.
```

Reply:
That's good to know thank you. Since this is more of a nice to have though, I'm going to just leave this as a follow up task because of the upcoming code freeze